targettests/execution/state_preparation.cpp

```diff
@@ -10,6 +10,7 @@
 // RUN: nvq++ %cpp_std --enable-mlir  %s                             -o %t && %t | FileCheck %s
 
 // Quantum emulators
+// RUN: nvq++ %cpp_std --target braket                   --emulate %s -o %t && %t | FileCheck %s
```

Comment:
This will need the `if $braket_avail;` wrapper when Bettina is ready to merge her PR. Can you flag that in her PR? Thanks.

Reply:
this one is running with the `--emulate` flag, do we still need the `if $braket_avail` test? It runs on my machine without any credentials.

Reply:
looked ad Bettina's PR, seems like it is still needed. Will flag.